### PR TITLE
Graphical query builder v0.9.0

### PR DESCRIPTION
**Pull request by Maxzor**

### CODE REVIEW

**Comment by Maxzor** [comment]
setup.sh now does the job :)

File: graphicalquerybuilder/dreyfus_wagner.hpp [hunk 1]
```diff
@@ -0,0 +1,318 @@
+//=======================================================================
```
Comment (rkhaotix) [inline graphicalquerybuilder/dreyfus_wagner.hpp]: @Maxzor is this file really needed here? It seems a boost module or something like this...

Comment (Maxzor) [reply graphicalquerybuilder/dreyfus_wagner.hpp]: It is a paal file, I modified it, I made public the entire class because I wanted to access most properties from graphicalquerybuilderpath.cpp
The setup.sh script takes it and move it into paal to replace the existing.
If you have ideas I am all ears!
